install/install_openadapt.ps1

```diff
@@ -1,11 +1,12 @@
 # PowerShell script to pull OpenAdapt and install
 
 ################################   PARAMETERS   ################################
-
 # Change these if a different version of is required
```

Comment:
I believe this sentence isn't grammatically correct

Reply:
Addressed in: [9e67435](https://github.com/MLDSAI/OpenAdapt/pull/312/commits/9e6743592a06cd7f07b088dc4c57cb1620987283)